src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(value) {
+      this.result += value;
+    },
+    subtract(value) {
+      this.result -= value;
+    },
+    divide(value) {
```

Comment:
```suggestion
    add(value) {
      this.result += value;
    },
    
    subtract(value) {
      this.result -= value;
    },
    
    divide(value) {
```

next time pls add empty line between methods